Makefile

```diff
@@ -60,6 +60,8 @@ ENVTEST_K8S_VERSION = 1.29
 # Set minimum Go version
 GOTOOLCHAIN_VERSION ?= go1.21.0
 
+ENABLE_WEBHOOKS ?= true
```

Comment:
This default is set to true? Where do we set it to  false? Is this about not including the webhook definitions in the bundle or not registering them with the controller manager?